Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -mod=vendor -a
 
 #####################################################################################################
 # Build the operator image
-FROM quay.io/openshift/origin-cli-artifacts:4.16 AS origincli
+FROM registry.redhat.io/openshift4/ose-cli-artifacts:latest AS origincli
```

Comment:
I'll let @rauhersu or @jc-rh confirm this ^

Reply:
yeah, @donpenney , you are right: main will always be latest, so we can keep this as it is for now. Your 4.15 branch, if it requires this image, will use 4.15.

Let me be extra verbose, though: the image you are using has the following tags : https://catalog.redhat.com/software/containers/openshift4/ose-cli-artifacts/5cdda1cad70cc57c44b2d7a5?architecture=amd64&image=65c58344ee15e43143521b94&container-tabs=overview

As you can see, latest = 4.15, but sooner or later, us and them will branch out for 4.16. At that moment, we should be using 4.16 here instead, and leave this 'latest' for main exclusively.

I do not know what is the best way for you to remind this: keep a Jira open ? I am gonna use a MR on the midstream side for this very reason, so prior to GA I will review ibu MR queues and will notice to merge the 4.16 reference instead of latest.

Reply:
Does 4.15 not work right now? If it works, why do we make work for ourselves when branching for 4.16?

Reply:
@jc-rh looks like the `latest` in this case `4.15`(no 4.16 tags)...
Yeah remembering to keep this would be painful. I wonder if you can automate that somehow or just always stick to latest (regardless of release or backport)

Reply:
/hold 

I am exploring another avenue to manage a 4.16 image ...